### ISSUE TITLE
babel plugin parallel？

### ISSUE THREAD

**Issue by mayu888** [webpack/webpack#12137]
My own Babel plug-in runs in parallel with other plug-ins. How can I run other plug-ins after all my Babel plug-ins have been run?

**Comment by mayu888** [comment]
> * [ ]  webpack-4
> * [ ]  question

**Comment by sokra** [comment]
It looks like you just deleted/ignored our lovely crafted [issue template](https://raw.githubusercontent.com/webpack/webpack/master/.github/ISSUE_TEMPLATE.md). It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please open a new issue with filled issue template. Also make sure your issue is not a question. Questions should be posted on Stack Overflow.
